src/components/Dashboard/TodoList.vue

```diff
@@ -100,22 +100,15 @@
                   <ContactInfo class="mt-4 contact-info-warning" />
                 </div>
 
-                <!-- alteration to a BEN in good standing -->
-                <div v-else-if="item.isAlteringToBen && isGoodStanding"
+                <!-- draft alteration to a BEN in good standing -->
+                <div v-else-if="isStatusDraft(item) && item.isAlteringToBen && isGoodStanding"
```

Comment:
This should be symmetric with line 83 above.

Once it's no longer a draft, we don't want to show this block; it will be handled like any other todo item.

Reply:
Screenshot - ULC alteration to BEN in good standing:

![image](https://user-images.githubusercontent.com/10002889/207674852-a4fa0c3d-9144-4174-918b-c1720b8b64d5.png)

Reply:
Screenshot - ULC alteration to BEN not in good standing:

![image](https://user-images.githubusercontent.com/10002889/207675421-b3e43476-385d-4efa-bf03-363391ba58a5.png)

Reply:
Screenshot - normal draft alteration:

![image](https://user-images.githubusercontent.com/10002889/207675719-bce4bbd3-51f5-497e-ae20-2874a1bd09aa.png)